src/lib/QtGui/QStyle.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, NativeElement } from '../core/Component';
+import { checkIfNativeElement } from '../utils/helpers';
+
+export class QStyle extends Component {
```

Comment:
Looks okay at the moment. But since QStyle https://doc.qt.io/qt-5/qstyle.html
inherits from QObject we will need to in future refactor it to match that so that those features are also available.